kcharles52/kt20-event-booking
# chore(setup): setup express app

**What does this PR do?**

- It enables the creation of an express application

**Description of Task to be completed?**

- install packages for application setup

**How should this be manually tested?**

- Run the following command
- `git fetch setup-app`
- `git checkout setup-app`
- `node app.js`
- The application will run 
- Navigate to `localhost:5001`
- You will see a message `This is the initial page`

**Any background context you want to provide?**

- Setup a graphql API needes to be setup on top of an express application or node application hence the need to setup the application first before setup the API

**Screenshots (if appropriate)**

- None

**Questions:**

- None
